CONTRIBUTING.md

```diff
@@ -304,8 +304,14 @@ VSCode     | [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeum
 IntelliJ   | Settings » Languages & Frameworks » JavaScript » Code Quality Tools » ESLint
 `vi`       | [scrooloose/syntastic](https://github.com/scrooloose/syntastic)
 
+#### Tools for Consistent Coding Style
+
 Another tool we use for enforcing consistent coding style is EditorConfig, which can be set up by installing a plugin in your editor that dynamically updates its configuration. Take a look at the [EditorConfig](http://editorconfig.org/#download) site to find a plugin for your editor, and browse our [`.editorconfig`](https://github.com/elastic/kibana/blob/master/.editorconfig) file to see what config rules we set up.
 
+Currently, we're migrating our codebase over to [Prettier](https://prettier.io/). Prettier is an opinionated code formatter that integrates with most editors. Prettier formats code when you save a file so that we spend less time discussing coding style and focus more on features and bugs. You can learn how to set up Prettier on your favorite editor by visiting [Prettier](https://prettier.io/) website. Our prettier rules can be found in [`.prettierrc`](https://github.com/elastic/kibana/blob/master/.prettierrc) file. 
```

Comment:
The correct way to setup prettier is to not setup prettier, but instead setup eslint, like @jbudz mentioned